### PR TITLE
Fix misleading string for dotnet add package

### PR DESCRIPTION
**Pull request by mairaw**
I thought that CmdPackageDescription was a bit misleading since the command only accepts one package. 
Since I was looking at the file, also reviewed the other strings for this command. 

I still want to do a more thorough string review as indicated in issue #5737, but I thought I'd forget about that specific misleading string when I finally get to that.

### CODE REVIEW

**Comment by mairaw** [comment]
Also wondering if these two strings should be in singular form: 
CmdSourceDescription - Uses a specific NuGet package source...
CmdPackageDirectoryDescription - Restores the package...

**Comment by eerhardt** [comment]
Looks like the test failures are real:

```
17:32:48                      Failed   Microsoft.DotNet.Cli.Package.Add.Tests.GivenDotnetPackageAdd.WhenMultiplePackagesArePassedCommandFails (TaskId:2)
17:32:48 17:32:48.442   4:2>DOTNETTEST : error Message:  [D:\j\workspace\release_windo---f861f43b\build\test\RunTest.proj]
17:32:48                       Expected string "Please specify only one package reference to be added." to contain "Please specify one package reference to add.". (TaskId:2)
```

@livarcocc @piotrpMSFT - can you get someone to check it out and fix?

**Comment by mairaw** [comment]
I can fix the [test file](https://github.com/dotnet/cli/blob/85ca206d84633d658d7363894c4ea9d59e515c1a/test/dotnet-add-package.Tests/GivenDotnetPackageAdd.cs) too if you guys agree with the change.

**Comment by livarcocc** [comment]
@rohit21agrawal Was there any reason behind the plural in the description of the parameter? Are you guys going to support multiple packages in this command?

**Comment by rohit21agrawal** [comment]
No, looks like a typo.
CC: @mishra14

**Comment by mairaw** [comment]
We've already merged the misleading string with @spboyer's PR: https://github.com/dotnet/cli/pull/5741

Things have been so crazy that I forgot that I had even reviewed that PR. But I have other improvements here as well.

**Comment by mishra14** [comment]
@mairaw @spboyer : Thanks for fixing this. 

This was a typo: `dotnet add package` only accepts one package at a time.

File: src/dotnet/commands/dotnet-add/dotnet-add-package/LocalizableStrings.cs [hunk 1]
```diff
@@ -9,21 +9,21 @@ internal class LocalizableStrings
...
+        public const string SpecifyExactlyOnePackageReference = "Please specify only one package reference to be added.";
 
-        public const string CmdFrameworkDescription = "Add reference only when targeting a specific framework";
+        public const string CmdFrameworkDescription = "Adds reference only when targeting a specific framework.";
 
-        public const string CmdNoRestoreDescription = "Add reference without performing restore preview and compatibility check.";
+        public const string CmdNoRestoreDescription = "Adds reference without performing restore preview and compatibility check.";
 
-        public const string CmdSourceDescription = "Use specific NuGet package sources to use during the restore.";
+        public const string CmdSourceDescription = "Uses specific NuGet package sources to use during the restore.";
```
Comment (livarcocc) [inline src/dotnet/commands/dotnet-add/dotnet-add-package/LocalizableStrings.cs]: I think this string still needs work. Uses specific... to use seems repeated.

**Comment by livarcocc** [comment]
@mairaw Please go ahead and fix the tests. I also commented on one string which I think we can still improve. Once the PR is green, we can take it.

File: src/dotnet/commands/dotnet-add/dotnet-add-package/LocalizableStrings.cs (hunk 1 above)
Comment (mairaw) [inline src/dotnet/commands/dotnet-add/dotnet-add-package/LocalizableStrings.cs]: True @livarcocc. How about "Specifies a NuGet package source to use during the restore."? 
It's only one source, right?

File: src/dotnet/commands/dotnet-add/dotnet-add-package/LocalizableStrings.cs [hunk 2]
```diff
@@ -9,21 +9,21 @@ internal class LocalizableStrings
...
 
-        public const string CmdPackageDirectoryDescription = "Restore the packages to this Directory .";
+        public const string CmdPackageDirectoryDescription = "Restores the packages to the specified directory.";
```
Comment (mairaw) [inline src/dotnet/commands/dotnet-add/dotnet-add-package/LocalizableStrings.cs]: should this also be singular?

Comment (livarcocc) [inline src/dotnet/commands/dotnet-add/dotnet-add-package/LocalizableStrings.cs]: No. Plural is fine. 
Also, for the sources above, I think it should also be plural.
@rohit21agrawal and @mishra14 Should sources be singular or plural?

**Comment by mairaw** [comment]
@livarcocc addressed your feedback. How will I make the tests pass since they currently expect a different string?

**Comment by livarcocc** [comment]
@mairaw Just update the strings expected by the tests.

**Comment by mairaw** [comment]
Ah I see, there are two test cases @livarcocc. In that case, I'll change that string a bit because that sentence only makes sense when you submit multiple packages, but not when you don't submit any. Unless we wanted to distinguish those scenarios in the command program.

**Pull request merged**